lib/jsdom/living/svg/SVGAnimatedString-impl.js

```diff
@@ -4,14 +4,22 @@ class SVGAnimatedStringImpl {
   constructor(globalObject, args, privateData) {
     this._element = privateData.element;
     this._attribute = privateData.attribute;
-    this._attributeDeprecated = privateData.attributeDeprecated; // can be undefined
+
+    if (privateData.attributeDeprecated) { // can be undefined
```

Comment:
This seems a bit unnecessary. Are there cases where `privateData.attributeDeprecatedNamespace` will be non-undefined, but `privateData.attributeDeprecated` will be undefined? If there are no such cases, then we could just keep this as two assignments, right?

Reply:
Ah yeah, this was a remnant from a previous iteration of the commit. The `if` isn't needed anymore.